cpplint.py

```diff
@@ -1111,21 +1111,18 @@ def ProcessCategory(category):
           error(filename, linenum, 'readability/nolint', 5,
                 f'Unknown NOLINT error category: {category}')
 
-def ProcessGlobalSuppresions(lines):
-  """Deprecated; use ProcessGlobalSuppressions."""
-  ProcessGlobalSuppressions(lines)
-
-def ProcessGlobalSuppressions(lines):
+def ProcessGlobalSuppressions(filename, lines):
```

Comment:
Should we start to add type hints to function parameters at some point?

Reply:
What I've been doing is adding type hints whenever I change a function's signature or code a new function. I should've added type hints here, but no matter.

Reply:
```suggestion
def ProcessGlobalSuppressions(filename: str, lines: list[str]) -> None:
```